src/java.desktop/share/legal/freetype.md

```diff
@@ -1,4 +1,4 @@
-## The FreeType Project: Freetype v2.12.0
+## The FreeType Project: Freetype v2.12.1
 
```

Comment:
Why does this show a .0 -> .1 update ?
The currently checked in file was updated in July so it already has .1 .. so that is very puzzling to me.

Reply:
Hmm .. it has disappeared with your 2nd commit. I can only suppose you were using a VERY out of date repo ..